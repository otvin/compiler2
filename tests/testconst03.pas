program testconst03(output);
{define a local constant using a global constant}

const globaltwo = 2;

procedure addtwo(a:integer);
const localtwo = globaltwo;
begin
	writeln(a + localtwo);
end;

begin
	writeln('should print 19.');
	addtwo(17);
end.
