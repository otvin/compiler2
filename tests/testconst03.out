should print 19.
19
